database/mysql/mysql_user.py

```diff
@@ -132,6 +147,12 @@
 '''
 
 EXAMPLES = """
+# Removes anonymous user account for localhost (the name parameter is required, but ignored)
+- mysql_user: name=anonymous user_anonymous=yes host=localhost state=absent
+
+# Removes all anonymous user accounts
+- mysql_user: name=anonymous user_anonymous=yes host_all=yes state=absent
+
```

Comment:
Good examples, just replace with name='' instead of name=anonymous user_anonymous=yes (assuming I am reading this right)